app/commands/user/insiders_status/update.rb

```diff
@@ -26,8 +26,8 @@ def update_eligible_lifetime
     when :active_lifetime
       user.update(insiders_status: :active_lifetime)
     when :active
-      user.update(insiders_status: :active_lifetime)
-      User::Notification::Create.(user, :joined_lifetime_insiders)
+      user.update(insiders_status: :eligible_lifetime)
+      User::InsidersStatus::Activate.(user)
```

Comment:
I'm using `User::InsidersStatus::Activate` to actually convert the `active` status to `active_lifetime`. That way we can have all activation-related logic (like awarding the badge) in one place